source/SIL.AppBuilder.Portal/src/lib/timeUtils.ts

```diff
@@ -31,9 +31,9 @@ export function getRelativeTime(date: Date | null) {
   return 'ERROR';
 }
 export function getTimeDateString(date: Date | null) {
-  return `${date?.toLocaleDateString() ?? '-'} ${
+  return `${date?.toLocaleDateString(languageTag()) ?? '-'} ${
```

Comment:
Do we want to use this or no? I have no idea whether this would ultimately be more efficient or not.

Reply:
I don't see efficiency being an issue here. I like having `languageTag()` here; very good.